cloudigrade/internal/viewsets.py

```diff
@@ -93,13 +93,33 @@ def perform_destroy(self, instance):
         tasks.delete_cloud_account.delay(instance.id)
 
 
-class InternalUserViewSet(InternalViewSetMixin, viewsets.ReadOnlyModelViewSet):
-    """Retrieve or list Users for internal use."""
+class InternalUserViewSet(
+    InternalViewSetMixin,
+    mixins.CreateModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
+    viewsets.ReadOnlyModelViewSet,
```

Comment:
I think this could be simplified since it's no longer read-only:
```suggestion
    viewsets.ModelViewSet,
```
See also https://github.com/encode/django-rest-framework/blob/master/rest_framework/viewsets.py#L235

Reply:
👍